language/en-GB/en-GB.site_menuitems.ini

```diff
@@ -11,3 +15,29 @@ MENU_DOWNLOADS_ITEM5="Language Packs"
 MENU_DOWNLOADS_ITEM6="Technical Requirements"
 MENU_DOWNLOADS_ITEM7="FAQs"
 MENU_DOWNLOADS_ITEM8="API Documentation"
+
+; Menus: Page Titles, Headings and Meta Descriptions
+MENU_DOWNLOADS_HOME_PAGETITLE="Build your website with the CMS Joomla!"
+MENU_DOWNLOADS_HOME_METADESC="Download the free open source CMS Joomla! and start building your awesome website."
+MENU_DOWNLOADS_LATEST_PAGETITLE="Joomla! Downloads - Download the latest version of the CMS"
+MENU_DOWNLOADS_LATEST_PAGEHEADING="Latest Joomla! Release"
+MENU_DOWNLOADS_LATEST_METADESCRIPTION="Download the latest release of Joomla. Update your site or build a new one."
+MENU_DOWNLOADS_DOWNLOADS_PAGETITLE="Joomla! Downloads - All releases of the open source CMS Joomla!"
+MENU_DOWNLOADS_DOWNLOADS_PAGEHEADING="All Joomla! Downloads"
+MENU_DOWNLOADS_DOWNLOADS_METADESC="Browse and download the Joomla! version you're looking for. All versions are available, even legacy versions."
+MENU_DOWNLOADS_EXTENSIONS_PAGETITLE="Download Official Joomla! Extensions: weblinks, install from web..."
+MENU_DOWNLOADS_EXTENSIONS_PAGEHEADING="Official Joomla! Extensions"
+MENU_DOWNLOADS_EXTENSIONS_METADESC="Browse and download the Joomla! Official Extensions you're looking for: Install from Web, Weblinks..."
+MENU_DOWNLOADS_LPACKS_PAGETITLE="Official Joomla! Language Packages"
+MENU_DOWNLOADS_LPACKS_METADESC="Browse and download the Joomla! Official Language Packages"
+MENU_DOWNLOADS_LPACKS_J4_PAGETITLE="Joomla 4 Translations: Download the language packages for your website"
+MENU_DOWNLOADS_LPACKS_J3_PAGETITLE="Joomla 3.x Translations: Download the language packages for your website"
+MENU_DOWNLOADS_LPACKS_J25_PAGETITLE="Joomla 2.5 Translations: Download the language packages for your website"
+MENU_DOWNLOADS_LPACKS_J15_PAGETITLE="Joomla 1.5 Translations: Download the language packages for your website"
+MENU_DOWNLOADS_LPACKS_J10_PAGETITLE="Joomla 1.0 Translations: Download the language packages for your website"
+MENU_DOWNLOADS_TECHREQ_PAGETITLE="Technical Requirements to run your Joomla! Website"
+MENU_DOWNLOADS_TECHREQ_PAGEHEADING="Technical Requirements"
+MENU_DOWNLOADS_TECHREQ_METADESC="Discover what are the technical requirements to run a website using the CMS Joomla, depending on its release."
```

Comment:
`MENU_DOWNLOADS_TECHREQ_METADESC="Discover the technical requirements to run a website using the Joomla, depending on its release."`